utils/libmarias3/CMakeLists.txt

```diff
@@ -1,21 +1,5 @@
 set(S3API_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libmarias3 CACHE INTERNAL "S3API_DIR")
 
-find_package(Git QUIET)
-
-if(GIT_FOUND AND EXISTS ${ENGINE_SRC_DIR}/.git)
-# Update submodules as needed
-    option(GIT_SUBMODULE "Check submodules during build" ON)
-    if(GIT_SUBMODULE)
-        message(STATUS "Submodule update")
-        execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init --recursive
-            WORKING_DIRECTORY ${ENGINE_SRC_DIR}
-                        RESULT_VARIABLE GIT_SUBMOD_RESULT)
-        if(NOT GIT_SUBMOD_RESULT EQUAL "0")
-            message(FATAL_ERROR "git submodule update --init failed with ${GIT_SUBMOD_RESULT}, please checkout submodules")
-        endif()
-    endif()
-endif()
-
```

Comment:
I don't see a replacement for this block of code in the new version.  Is submodule pulling a manual step in the community server?

Reply:
No, it's done by `cmake/submodules.cmake` for all submodules, so your separate `git submodule update` is redundant.

_the server has, like, seven (?) submodules now, and after the second submodule we've figured out that if all submodules will need this auto-updating code, we can as well put it in the server and spare submodules some troubles and avoid some copy-pasting_